### ISSUE TITLE
Check that project

### ISSUE THREAD

**Issue by shapkarin** [koolamusic/awesome-imba#27]
I found and read that project https://github.com/athif23/start-imba not run in yet, but feels awesome )

**Comment by koolamusic** [comment]
Wow, seen that, should we wait till they publish an npm package ?

**Comment by shapkarin** [comment]
Not sure why they does not publish it.. I’ll check it too and try to ask about npm

**Comment by shapkarin** [comment]
Not sure why they does not publish it.. 
ref: https://github.com/athif23/start-imba/issues/2

**Comment by koolamusic** [comment]
Do you think we should add It, hopefully someone can notice it from our repo on contribute

**Comment by shapkarin** [comment]
I still don’t have time to check that closely. I think it’s should If it’s works fine.

**Comment by scanf** [comment]
I have used athif23/start-imba and it works fine even though it lacks some features. From the explanation in the [README file](https://github.com/athif23/start-imba#how-to-use-and-contributing) it's easy to install. I used it to build https://xiyo.no/

Maybe the author is just busy and has not gotten around to the things todo. Would definitely recommend adding it to the awesome list since it setups the boilerplate for you :)
